doc/conf.py

```diff
@@ -32,16 +34,36 @@
 # ones.
 extensions = [
     'sphinx.ext.autodoc',
+    'sphinx.ext.autosummary',
     'sphinx.ext.mathjax',
     'sphinx.ext.viewcode',
     'sphinx.ext.napoleon',  # support for NumPy-style docstrings
     'sphinx.ext.intersphinx',
+    'sphinx.ext.doctest',
+    'sphinxcontrib.bibtex',
+    'sphinx.ext.extlinks',
     'matplotlib.sphinxext.plot_directive',
+    'nbsphinx',
+]
+
+bibtex_bibfiles = ['references.bib']
+
+nbsphinx_execute_arguments = [
+    "--InlineBackend.figure_formats={'svg', 'pdf'}",
+    "--InlineBackend.rc={'figure.dpi': 96}",
 ]
 
+# Tell autodoc that the documentation is being generated
+sphinx.SFA_DOCS_ARE_BEING_BUILT = True
```

Comment:
This makes only sense if you are actually using it, see

https://github.com/sfstoolbox/sfs-python/blob/2391e1edeeab522abb62122692b34021e6f94045/sfs/__init__.py#L54-L57